docs/Configuration.md

```diff
@@ -263,6 +254,44 @@ Custom Annotation names may also be configured using ConfigMap Data Values.
 
 Note: The requirement to label the build with `app.kubernetes.io/name=<app_name>` for the annotated Builds applies.
 
+#### An example workflow for an OpenShift binary build:
+
+```
+# sample_app in current directory
+cat app.py 
+#!/usr/bin/env python3
+print("Hello World")
```

Comment:
Can we make this two different snippets to reduce confusion?

<pre>
```python
# app.py contents here
```
</pre>
<pre>
```bash
# script contents here
```
</pre>

Reply:
aye.. thank you!